### ISSUE TITLE
Putty based software cannot access agent

### ISSUE THREAD

**Issue by Korkman** [buptczq/WinCryptSSHAgent#47]
I have a weird problem on one machine. The WinCryptSSHAgent is running and working fine for WSL, but pageant based clients fail. They do recognize WCSA as the agent, but fail to obtain the keys. Running the original pageant, they work.

Event log from putty:
```
Pageant is running. Requesting keys.
Failed to get reply from Pageant.
```

The machine has Sophos Endpoint Protection 10.8 installed by the employer. I was able to add both putty and WinCryptSSHAgent to several whitelists, but nothing helps. I verified no other pageant was running while WCSA was running. Tried running with elevated privileges, both WCSA and putty. WinSCP shows the same problem.

Any ideas? Is Sophos the right direction, or might there be some Windows 10 settings interrupting the communication? Windows Event log shows nothing ...

Edit: tried both 32-bit and 64-bit versions of both putty and WCSA as well
Edit 2: and also a freshly generated openssh key with default settings

**Comment by buptczq** [comment]
I added some debug logs to WCSA, please use this version and collect debug information:
1. Download https://github.com/buptczq/WinCryptSSHAgent/releases/download/v1.1.8-debug/WinCryptSSHAgent.exe
2. Run `setx WCSA_DEBUG 1` and reboot to take effect
3. Reproduce your problem
4. Upload debug log in `%USERPROFILE%\WCSA_DEBUG.log`

**Comment by Korkman** [comment]
```
Bei einem Socketvorgang war das Netzwerk inaktiv.
Bei einem Socketvorgang war das Netzwerk inaktiv.
Pageant: OpenFileMapping 670956517104 23 PageantRequest00001b6c
Pageant: OpenFileMapping error Der Vorgang wurde erfolgreich beendet.
```
The german bits translated:
```
During a socket operation the network was inactive.
During a socket operation the network was inactive.
Pageant: OpenFileMapping 670956517104 23 PageantRequest00001b6c
Pageant: OpenFileMapping error Task completed successfully.
```

**Comment by buptczq** [comment]
Try pre-released v1.1.9 please. If it works, please tell me.

https://github.com/buptczq/WinCryptSSHAgent/releases/download/v1.1.9/WinCryptSSHAgent.exe

**Comment by Korkman** [comment]
Yes, it works! Thank you very much for your efforts.
